Makefile

```diff
@@ -37,10 +37,10 @@ src/pykcs11_wrap.cpp-py3: src/pykcs11.i
 src/pykcs11.i: src/opensc/pkcs11.h src/pkcs11lib.h src/pykcs11string.h src/ck_attribute_smart.h
 	touch $@
 
-dist: clean
+dist: src/pykcs11_wrap.cpp-py2 src/pykcs11_wrap.cpp-py3 clean
```

Comment:
Be careful. make will generate the 3 targets in parallel if `-j` is used.
Maybe what you want, in fact, is:
```
dist: src/pykcs11_wrap.cpp-py2 src/pykcs11_wrap.cpp-py3
    $(MAKE) clean
```